guides/source/upgrading_ruby_on_rails.md

```diff
@@ -147,18 +147,18 @@ documentation.
 
 ### Autoloading is Disabled After Booting in the Production Environment
 
-Autoloading is now disabled after booting in the production environment by
-default.
-
-Eager loading the application is part of the boot process, so top-level
-constants are fine and are still autoloaded, no need to require their files.
-
-Constants in deeper places only executed at runtime, like regular method bodies,
-are also fine because the file defining them will have been eager loaded while booting.
-
-For the vast majority of applications this change needs no action. But in the
-very rare event that your application needs autoloading while running in
-production mode, set `Rails.application.config.enable_dependency_loading` to
+Autoloading of paths appended to `config.autoload_paths` is now disabled after
+booting in the production environment by default. Eager loading the application
```

Comment:
I'll make it a new paragraph, it's not a continuation of the previous sentence and new lines are ignored in Markdown so if you want a separation of these points they should be a new paragraph. 

Is that all right?